PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/CONTRIBUTING.MD

```diff
@@ -0,0 +1,82 @@
+### Configuring MS Application Insights
```

Comment:
Don't think this needs to be in a contributing.md can we just append this to the original readme we have or a separate, "Configure Telemetry" document.

Reply:
I can rename to ConfigureTelemetry.md I think it's easier to keep them separate